mod/05_email/format/email_address_format.rb

```diff
@@ -0,0 +1,15 @@
+# -*- encoding : utf-8 -*-
+
+class Card::EmailAddressFormat < Card::TextFormat  
+  def link_to  text, href, opts={}
+    if text and href != text
+      "#{text} <#{href}>"
+    else
+      href
+    end
+  end
+
+  def chunk_list
+    :references
+  end
+end
```

Comment:
Another point, automatically adding mailto: or http: to the link part of a chunk reference should really be part of rendering the link and not in the chunk processing.  In a way, that is the origin of this bug.  When generating an html link, it is needed on the href= attribute, but that is format dependent and should not be in the chunk processing.